source/common/access_log/access_log_manager_impl.cc

```diff
@@ -42,7 +42,8 @@ AccessLogFileImpl::AccessLogFileImpl(Filesystem::FilePtr&& file, Event::Dispatch
 }
 
 void AccessLogFileImpl::open() {
-  const Api::IoCallBoolResult result = file_->open();
+  static constexpr Filesystem::FlagSet flags{0xf};
```

Comment:
This is a bit too magical. Can you make it more clear what flags are being used by default here? Perhaps there should be some type of defaultFlags() function somewhere or similar?